### PR TITLE
SLT and SLTU instructions

### PR DESCRIPTION
**Pull request by hanstig**
Added the slt and sltu instructions as well as some very basic tests for both.

### CODE REVIEW

**Review by eim64 — CHANGES_REQUESTED** [review]
Using outdated RegisterFile will not compile on main.

**Comment by hanstig** [comment]
The issues should be fixed now

File: tests/executor.cpp [hunk 1]
```diff
@@ -226,3 +226,71 @@ TEMPLATE_TEST_CASE("lui", "[Executor]", RegisterFile32, RegisterFile64) {
 		REQUIRE(reg_file.get(RegisterName::e_t1).u == 0xbeef0000);
 	}
 }
+
+TEST_CASE("slt", "[Executor]") {
+    SECTION("Equal numbers") {
+        using Address = typename RegisterFile::Unsigned;
```
Comment (eim64) [inline tests/executor.cpp:232]: Is Adress even used? if not, you should just get rid of this.

**Review by eim64 — CHANGES_REQUESTED** [review]
I recommend rebasing and force pusing so that only relevant commits remain. (no quick fix commits for instance) so that the commit history stays clean.

**Comment by hanstig** [comment]
Unnecessary code removed, but I'm not sure how to force push and rebase in in order to fix the commit messages?

**Review by eim64 — APPROVED** [review]

**Pull request merged**